mozilla_vpn/views/active_subscriptions_table.view.lkml

```diff
@@ -27,7 +27,12 @@ view: +active_subscriptions_table {
     description: "Get max active date from end date in active date filter.  If null, use last modified date."
     hidden: yes
     type: date
-    sql: COALESCE({% date_end active_date %}, ${metadata.last_modified_date})-1;;
+    sql: CASE
+    WHEN
+    ifnull({% date_end active_date %}, ${metadata.last_modified_date})<${metadata.last_modified_date}
+    THEN {% date_end active_date %}
+    ELSE ${metadata.last_modified_date}
+    END-1 ;;
```

Comment:
This can more succinctly be described with `LEAST`:
```suggestion
    sql: LEAST(
      IFNULL({% date_end active_date %}, ${metadata.last_modified_date}),
      ${metadata.last_modified_date}
    )-1 ;;
```